### DISCUSSION TITLE
Fix everything in the doc script for `package.json`

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#24405] [Suggest an Idea]
### Tell us more.

This feature is for the Renovate repository itself, _not_ for the Renovate program.

We now have a `"lint-documentation"` script, that can check if I'm making bad Markdown links again. 😄 But it's easy to forget to run the link check. I end up running `prettier-fix` only and then commit and push. 🙈 

I'd like a one-stop-command that runs:

- `lint-documentation`
- `doc-fence-check`
- Prettier in fixing mode
- Markdown lint in fix mode
- Anything else that I'm forgetting here

I'm thinking of a name like `doc-fix-everything`, or at least something short, and easy to remember.

**Comment by HonkingGoose** [comment]
@rarkins what do you think of this idea?

**Reply by rarkins** [reply]
Sure, sounds fine

**Reply by HonkingGoose** [reply]
Discussion resolved with:

- #24482

**Comment by majed08** [comment]
👌

**Reply by majed08** [reply]
👌
